.github/workflows/check_issue_label.yml

```diff
@@ -0,0 +1,115 @@
+
+name: Check Issue Labels
+
+on:
+  issues:
+    types:
+      - labeled
+
+env:
+  PAT_TOKEN: ${{ secrets.TEAM_READ_PAT }}  # PAT with read:org scope
+
+permissions: {}
+
+jobs:
+  check-label:
+    runs-on: ubuntu-22.04
+    permissions:
+      issues: write
+
+    steps:
+      - name: Validate issue label
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ env.PAT_TOKEN }}
```

Comment:
you can just reference `secrets.TEAM_READ_PAT` here and skip passing the token through `env`